apps/arena/priv/config.json

```diff
@@ -1167,28 +1167,54 @@
       "radius": 200.0,
       "effects": [
         "golden_clock_effect"
-      ]
+      ],
+      "mechanics": {}
     },
     {
       "name": "magic_boots",
       "radius": 200.0,
       "effects": [
         "magic_boots_effect"
-      ]
+      ],
+      "mechanics": {}
     },
     {
       "name": "mirra_blessing",
       "radius": 200.0,
       "effects": [
         "mirra_blessing_effect"
-      ]
+      ],
+      "mechanics": {}
     },
     {
       "name": "giant",
       "radius": 200.0,
       "effects": [
         "giant_effect"
-      ]
+      ],
+      "mechanics": {}
+    },
+    {
+      "name": "fake_item",
+      "radius": 200.0,
+      "effects": [],
+      "mechanics": {
+        "spawn_bomb": {
+          "name": "bomb",
+          "radius": 200.0,
+          "preparation_delay_ms": 2000,
+          "activation_delay_ms": 3000,
+          "vertices": [],
+          "activate_on_proximity": true,
+          "mechanics": {
+            "circle_hit": {
+              "damage": 64,
+              "range": 380.0,
+              "offset": 400
+            }
+          }
+        }
+      }
```

Comment:
I think we should we remove this until the client spawn the trap object in the map

Reply:
Yes, I agree too, before merging it we have to delete it

Reply:
Please remove this then we're good to go.